cookbook/AutoEval_Experiments.ipynb

```diff
@@ -0,0 +1,858 @@
+                    "output_type": "execute_result"
+                }
+            ],
+            "source": [
+                "from lastmile.lib.auto_eval import BuiltinMetrics\n",
+                "default_metrics = [\n",
+                "    BuiltinMetrics.FAITHFULNESS,\n",
+                "    BuiltinMetrics.RELEVANCE,\n",
+                "    # LastMile provides a few other metrics out of the box such as Toxicity, Question-Answering, etc.\n",
+                "]\n",
+                "\n",
+                "print(f\"Evaluation job kicked off\")\n",
+                "evaluation_results = client.evaluate_dataset(\n",
+                "    dataset_id=dataset_id,\n",
+                "    metrics=default_metrics,\n",
+                "    # Log this evaluation run to the experiment we created above\n",
+                "    experiment_id=experiment.id,\n",
+                "\n",
+                "    # You can add additional metadata properties here, which will get merged with any metadata on the Experiment itself\n",
+                "    # The metadata properties are surfaced as columns in the resulting dataset\n",
+                "    metadata={\n",
+                "        \"extras\": \"Base metric tests\"\n",
+                "    }\n",
+                ")\n",
+                "\n",
+                "print(\"Evaluation Results:\")\n",
+                "evaluation_results.head(10)"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "## 6. Visualize this in the Experiments Console\n",
+                "\n",
+                "You can log as many evaluation runs under an experiment as you want, and compare them.\n",
+                "The AutoEval UI has visualizations for this purpose:\n",
+                "\n",
+                "* Experiments: https://lastmileai.dev/evaluations?view=experiments\n",
+                "* Evaluation Runs: https://lastmileai.dev/evaluations?view=all_runs\n",
+                "* Project Dashboard: https://lastmileai.dev/dashboard\n",
+                "* Dataset Library: https://lastmileai.dev/datasets\n",
+                "\n",
+                "Check it out for yourself!"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "![Experiment Comparison](https://github.com/user-attachments/assets/8ccb3a2b-f1c5-4d69-9490-a00f385a7acc)"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "![Evaluation Run](https://github.com/user-attachments/assets/26d8f62d-0dcb-4c5f-b23e-2b3fa6e289b2)"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "![Project Dashboard](https://github.com/user-attachments/assets/7156d91f-9286-4996-989d-b0b76c623454)"
```

Comment:
For some reason these images aren't rendering in this notebook view -- https://github.com/lastmile-ai/lastmile-docs/blob/332d16f000a3769b9af2b5f5459474e57626ad68/cookbook/AutoEval_Experiments.ipynb